examples/chatbot/index.ts

```diff
@@ -83,9 +87,28 @@ const server = app.listen(PORT, async () => {
   )
 
   try {
+    /**
+     * Note: To test credential revocation locally, use the following configuration:
+     * const credentialDefinition = (await apiClient.credentialTypes.create({
+     *     id: randomUUID(),
+     *     name: "phoneNumber",
+     *     version: '1.0',
+     *     attributes: ['phoneNumber'],
+     *     supportRevocation: true
+     * }))
+     */
+   const credentialDefinition = (await apiClient.credentialTypes.import(phoneCredDefData))
     phoneNumberCredentialDefinitionId =
-      phoneNumberCredentialType?.id ?? (await apiClient.credentialTypes.import(phoneCredDefData)).id
+      phoneNumberCredentialType?.id ?? credentialDefinition.id
     logger.info(`phoneNumberCredentialDefinitionId: ${phoneNumberCredentialDefinitionId}`)
+    phoneNumberRevocationDefinitionId = 
+      phoneNumberCredentialType?.id ?
+        (await apiClient.revocationRegistry.get(phoneNumberCredentialDefinitionId))[0] :
+        await apiClient.revocationRegistry.create({
+          credentialDefinitionId: phoneNumberCredentialDefinitionId,
+          maximumCredentialNumber: 1000
+        })
```

Comment:
This logic takes for granted that if a phone number credential type is created, it does exist a revocation registry associated to it. It seems to me that you can it would be more correct and simpler something like:

```suggestion
    phoneNumberRevocationDefinitionId = 
        (await apiClient.revocationRegistry.get(phoneNumberCredentialDefinitionId))[0]  ??
        await apiClient.revocationRegistry.create({
          credentialDefinitionId: phoneNumberCredentialDefinitionId,
          maximumCredentialNumber: 1000
        })
```